api/rest/restcore/projects_rest.php

```diff
@@ -114,6 +122,121 @@ function rest_project_version_add( \Slim\Http\Request $p_request, \Slim\Http\Res
 	return $p_response->withStatus( HTTP_STATUS_NO_CONTENT, "Version created with id $t_version_id" );
 }
 
+/**
+ * A method to add a project subproject.
+ *
+ * @param \Slim\Http\Request $p_request   The request.
+ * @param \Slim\Http\Response $p_response The response.
+ * @param array $p_args Arguments
+ * @return \Slim\Http\Response The augmented response.
+ */
+function rest_project_subproject_add( \Slim\Http\Request $p_request, \Slim\Http\Response $p_response, array $p_args ) {
+	$t_project_id = isset( $p_args['id'] ) ? $p_args['id'] : $p_request->getParam( 'id' );
+	if( is_blank( $t_project_id ) ) {
+		$t_message = "Project id is missing.";
+		return $p_response->withStatus( HTTP_STATUS_BAD_REQUEST, $t_message );
+	}
+
+	$t_subproject_to_add = $p_request->getParsedBody();
+	$t_subproject_id_to_add = $t_subproject_to_add['subproject_id'];
+
+
+	$t_data = array(
+		'query' => array(
+			'project_id' => $t_project_id
+		),
+		'payload' => $t_subproject_to_add
+	);
+
+	$t_command = new SubprojectAddCommand( $t_data );
+	$t_subproject_added = $t_command->execute();
+
+	return $p_response->withStatus( HTTP_STATUS_CREATED,
```

Comment:
Adding a version returns and updating a sub-project returns no content status code and no json.  This seems inconsistent.